sdlib/d/gc/bin.d

```diff
@@ -24,7 +52,7 @@ struct Bin {
 
 	void* alloc(shared(Arena)* arena, shared(ExtentMap)* emap,
 	            ubyte sizeClass) shared {
-		assert(sizeClass < ClassCount.Small);
+		assert(isSmallSizeClass(sizeClass));
```

Comment:
You could be sending these type of changes in their own MR and they would get merged pretty much immediately. Don't bundle a ton of things together.